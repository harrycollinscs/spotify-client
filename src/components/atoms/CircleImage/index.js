import React from "react";
import styled from "styled-components";

const StyledImage = styled.img`
  width: 50px;
  height: 50px;
  border-radius: 50px;
  border: 1px solid white;
  margin: 0px 30px;
`;

const CircleImage = ({ url }) => <StyledImage src={url} />;

export default CircleImage;
