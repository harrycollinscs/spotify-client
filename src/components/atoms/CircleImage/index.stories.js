import React from "react";
import CircleImage from "./index";

export default {
  title: "Atoms/CircleImage",
  component: CircleImage,
  argTypes: {
    size: {
      options: ["small", "medium", "large"],
      control: { type: "select" },
    },
  },
};

const Template = (args) => <CircleImage {...args} />;

export const Default = Template.bind({});
Default.args = {
  url: "https://i.scdn.co/image/ab6761610000e5eb3ef779aa0d271adb2b6a3ded",
  size: "medium",
};



