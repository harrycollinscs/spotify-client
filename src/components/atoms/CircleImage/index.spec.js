import React from "react";
import CircleImage from "./index";
import { shallow } from "enzyme";

describe("Atoms/CircleImage", () => {
  it("Should match snapshot", () => {
    const render = shallow(<CircleImage />);
    expect(render).toMatchSnapshot();
  });
});
