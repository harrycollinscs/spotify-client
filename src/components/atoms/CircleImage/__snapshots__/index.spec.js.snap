// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/CircleImage Should match snapshot 1`] = `<styled.img />`;
